src/components/sidebar/routePathView/RoutePathViewForm.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import InputContainer from '../InputContainer';
+import { Button, Dropdown, Checkbox } from '../../controls';
+import ButtonType from '../../../enums/buttonType';
+import * as s from './routePathView.scss';
+
+interface IRoutePathViewFormState {
+}
+
+interface IRoutePathViewFormProps {
+    isEditingDisabled: boolean;
+}
+
+class RoutePathViewForm extends React.Component<IRoutePathViewFormProps, IRoutePathViewFormState>{
+    constructor(props: any) {
+        super(props);
+
+    }
+
+    // TODO
+    public onChange = () => {
+    }
+
+    public render(): any {
+        const isEditingDisabled = this.props.isEditingDisabled;
+
+        return (
+        <div>
```

Comment:
Well this is probably outside of the scope for this pr, but why do we not use `<form>` for form?

Reply:
No need to use form. Form typically uses "submit handlers"